notifications/messageTypes.ts

```diff
@@ -0,0 +1,9 @@
+export enum MessageType {
+    MESSAGE = 'MessageType.MESSAGE',
+    MATCH = 'MessageType.MATCH',
+    COURSE = 'MessageType.COURSE',
+    APPOINTMENT = 'MessageType.APPOINTMENT',
+    SURVEY = 'MessageType.SURVEY',
+    NEWS = 'MessageType.NEWS',
+    CHAT = 'MessageType.CHAT',
+}
```

Comment:
why `MessageType.MESSAGE` and not `message` as a string?
I need the following MessageTypes: **chat, match, course, appointment, survey, news**

Reply:
fixed